Rakefile

```diff
@@ -82,6 +82,8 @@ desc 'Build the Windows gRPC DLLs for Ruby'
 task 'dlls' do
   grpc_config = ENV['GRPC_CONFIG'] || 'opt'
   verbose = ENV['V'] || '0'
+  # use env variable to set artifact build paralellism, default to 4 cores
+  nproc_override = ENV['GRPC_RUBY_BUILD_PROCS'] || '4'
```

Comment:
To now limit ourselves to 4 cores, how about

```
ENV['GRPC_RUBY_BUILD_PROCS'] || `nproc`
```

?

Reply:
good idea. done.